app/cdap/components/Popover/index.js

```diff
@@ -81,10 +83,15 @@ export default class Popover extends PureComponent {
   };
 
   componentDidMount() {
+    this._isMounted = true;
```

Comment:
https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html We should try to avoid this. Seems like we are missing some to cancel or unsubscribe to some event handler that triggers a setState on an unmounted component.